.pre-commit-config.yaml

```diff
@@ -6,3 +6,10 @@ repos:
       - id: trailing-whitespace
       - id: end-of-file-fixer
       - id: check-added-large-files
+  - repo: https://github.com/dtaivpp/commit-msg-regex-hook
+    rev: v0.2.2
+    hooks:
+      - id: commit-msg-regex-hook
+        args: [ "--pattern='^(feat|chore|sec|fix|refactor|docs|build|ci|test)(\\([A-Z]{3,6}-[0-9]{3,6}\\))?: .+$'",
```

Comment:
Here is missing from CONTRIBUTING.md
- deps

Reply:
Added